cassandra/src/main/scala/akka/stream/alpakka/cassandra/CqlSessionProvider.scala

```diff
@@ -38,6 +38,8 @@ trait CqlSessionProvider {
  */
 class DefaultSessionProvider(system: ActorSystem, config: Config) extends CqlSessionProvider {
 
+  def this(system: ClassicActorSystemProvider, config: Config) = this(system.classicSystem, config)
```

Comment:
IIRC this is problematic for the use side as `ActorSystem` is a `ClassicActorSystemProvider`. The compiler can't choose.

Reply:
You're right. I did some testing locally and it wasn't allowed. I'll just replace the default constructor with `ClassicActorSystemProvider`.